.github/labels.yml

```diff
@@ -30,6 +31,8 @@
 - name: "Category: Label missing ❗"
   description: "Use this if it feels a label should be added to label this issue"
   color: "ffc7ea"
+- name: "Category: Question / Help wanted ❓"
+  from_name: "question"
```

Comment:
Do we want questions/help wanted as issues though? In my experience discussions are more appropriate for this (otherwise you just get flooded with issues asking for simple things - discussions are a bit more forum oriented and won't mix up with more 'serious'/developer-focused issues.

This made me think we should have a contact link in the issue creation process to redirect users needing help to use the github discussions, done in https://github.com/ava-labs/libevm/pull/96 (I also updated issue templates, which I think is worth the small conflict versus the base repo)

```suggestion
```